updated for new getting help process (#396)
smartcontractkit/documentation: docs/chainlink-keepers/faqs.md

## Will Chainlink Keepers work for my use case?

> We are here to help! For help with your use case, please [reach out directly](mailto:keeper@chain.link) and we would be happy to connect you with one of our Solutions Architects. You can also ask questions on Chainlink Keepers by asking on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=chainlink) or joining the [#keepers channel](https://discord.com/channels/592041321326182401/821350860302581771) in our [Discord server](https://discord.gg/qj9qarT).

Reviewer: ```suggestion
We are here to help! For help with your use case, please [reach out directly](mailto:keeper@chain.link) and we would be happy to connect you with one of our Solutions Architects. You can also ask questions about Chainlink Keepers on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=chainlink) or in the [#keepers channel](https://discord.com/channels/592041321326182401/821350860302581771) in our [Discord server](https://discord.gg/qj9qarT).
```
